04_pig_latin/pig_latin.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+def translate(string)
+  vowels = %w[a e i o u A E I O U]
```

Comment:
nah, `vowels = %w[a e i o u]` should be enough here
modify further code to use my version of this variable